src-docs/src/components/guide_section/_utils.test.js

```diff
@@ -314,5 +314,30 @@ describe('renderJsSourceCode', () => {
           export default () => 'Hello world!';`)
       );
     });
+
+    it('handles importing React correctly by ignoring code comments', () => {
```

Comment:
```suggestion
    it('does not handle import statements within template literal backticks', () => {
```

Test itself looks great!